source/features/rgh-feature-descriptions.tsx

```diff
@@ -49,6 +49,11 @@ async function init(): Promise<void | false> {
 					<div dangerouslySetInnerHTML={{__html: feature.description}} className="text-bold"/>
 					<div className="no-wrap">
 						<a href={conversationsUrl.href} data-pjax="#repo-content-pjax-container">Conversations</a>
+						{
+							location.pathname.endsWith('css')
```

Comment:
```suggestion
							location.pathname.endsWith('.css')
```

Reply:
This already applies to features exclusively, so unless we have an extensionless feature named `hellocss` this change isn't necessary